mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTInboundHandler.java

```diff
@@ -120,6 +122,17 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
         ctx.close();
     }
 
+    public void userEventTriggered(ChannelHandlerContext ctx, Object event) throws Exception {
+        if (event instanceof IdleStateEvent) {
```

Comment:
IMO, when event is not `IdleStateEvent` we better be let trigger the parent handler.
Example:
```java
if (event instanceof IdleStateEvent) {
    // handle IdleStateEvent
} else {
    super.userEventTriggered(ctx, event);
}
```

Reply:
Yes, right.